src/login/LoginView.js

```diff
@@ -202,24 +222,25 @@ export class LoginView {
 	}
 
 	_setupHooks() {
-		this.oncreate = () => {
-			// When iOS does auto-filling (always in WebView as of iOS 12.2 and in older Safari)
-			// it only sends one input/change event for all fields so we didn't know if fields
-			// were updated. So we kindly ask our fields to update themselves with real DOM values.
-			this._formsUpdateStream = stream.combine(() => {
-				requestAnimationFrame(() => {
-					this.mailAddress.updateValue()
-					this.password.updateValue()
-				})
-			}, [this.mailAddress.value, this.password.value])
-		}
-		this.onremove = () => {
-			this.password.value("")
-
-			if (this._formsUpdateStream) {
-				this._formsUpdateStream.end(true)
-			}
-		}
+		// TODO not sure if this is necessary when using TextFieldN in the LoginForm anyway
```

Comment:
I think it's still necessary otherwise we don't get update event for all fields. It's really nasty, DOM is updated but we don't know about it because event was not fired. We need some workaround for that, maybe we should grab DOM fields in `oncreate`

Reply:
Ok, we should move this into LoginForm then. Will have to go into the office to test it, I could do that tomorrow